2-Browsers/Week1/homework/ex1-bookList/index.js

```diff
@@ -19,7 +19,35 @@
 //cspell: enable
 
 function createBookList(books) {
-  // your code goes in here, return the ul element
+  const ulBook = document.createElement('ul');
+
+  books.forEach((book) => {
+    const liBook = document.createElement('li');
+    const parBook = document.createElement('p');
+    parBook.innerText = `${book.title} - ${book.author}`;
+
+    liBook.appendChild(parBook);
+
+    const imgBook = document.createElement('img');
+    if (book.title === 'The Design of Everyday Things') {
```

Comment:
This is not part of the assignment, but the best way to do this is to move the URLs to the `myBooks` array on line 53 so you don't need this if..else here.

Reply:
Hi Martijn;

Yes you are right and it becomes more easier and clean 
But what I understood from the exercise is that I have to code a function only but not allowed to change the array;
thats why I didn't touch the data in array